### PR TITLE
Add ClusterIssuerName configuration option

### PR DESCRIPTION
**Pull request by ubergesundheit**
This PR introduces a new configuration option to specify the name of the cert-manager ClusterIssuer to be used on Ingress resources installed by the chart in this repository.

Towards https://github.com/giantswarm/roadmap/issues/2844

## Checklist

I have:

- [x] Described why this change is being introduced
- [x] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

**Comment by QuentinBisson** [comment]
Is there a reason why we might need this?

**Comment by ubergesundheit** [comment]
> Is there a reason why we might need this?

cabbage recently updated cert-manager templates and configuration to follow upstream more closely. One of the changes is that cert-manager does not have a default issuer selected. This makes sure Ingresses can still obtain TLS certificates through cert-manager + let's encrypt by explicitly specifying which ClusterIssuer to use.

File: helm/prometheus-meta-operator/templates/alertmanager/alertmanager-ingress.yaml [hunk 1]
```diff
@@ -2,7 +2,12 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   annotations:
-    kubernetes.io/tls-acme: "{{ .Values.prometheus.letsencrypt }}"
+    {{- if .Values.prometheus.letsencrypt }}
```
Comment (QuentinBisson) [inline helm/prometheus-meta-operator/templates/alertmanager/alertmanager-ingress.yaml:5]: We have a few ingresses that are generated in PMO code. Could you also add it to them?

**Comment by QuentinBisson** [comment]
Thanks I just saw that in the roadmap issue because I did not check it prior to me asking the question :(

File: helm/prometheus-meta-operator/templates/alertmanager/alertmanager-ingress.yaml (hunk 1 above)
Comment (QuentinBisson) [reply helm/prometheus-meta-operator/templates/alertmanager/alertmanager-ingress.yaml:5]: Oh right because they rely on the main ones as they are subpaths

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**